openedx/core/djangoapps/catalog/admin.py

```diff
@@ -0,0 +1,14 @@
+"""
+Django admin pages for catalog support models.
+"""
+from django.contrib import admin
+
+from config_models.admin import ConfigurationModelAdmin
+from openedx.core.djangoapps.catalog.models import CatalogIntegration
+
+
+class CatalogIntegrationAdmin(ConfigurationModelAdmin):  # pylint: disable=missing-docstring
```

Comment:
I can do one better by just getting rid of this class. It's not actually necessary!